readme.md

```diff
@@ -83,37 +135,47 @@ Some text.
 MIT
 ```
 
+> 👉 **Note**: These contributors are inferred from this project’s
+> [`package.json`][package-json].
+> Running this example in a different package will yield different results.
+
 ## API
 
 This package exports no identifiers.
 The default export is `remarkContributors`.
 
 ### `unified().use(remarkContributors[, options])`
 
-Inject a given list of contributors.
+Generate a list of contributors.
+In short, this plugin:
 
-*   Looks for the first heading matching `/^contributors$/i` or `options.heading`
-*   If no heading is found and `appendIfMissing` is set, inject such a heading
-*   Replaces the table in that section if there is one, or injects it otherwise
+*   looks for the first heading matching `/^contributors$/i` or `options.heading`
+*   if no heading is found and `appendIfMissing` is set, injects such a heading
+*   if there is a heading, replaces everything in that section with a new table
 
-##### Options
+##### `options`
+
+Configuration (optional in Node.js, required in browsers).
 
 ###### `options.contributors`
 
 List of contributors to inject (`Array<Object>`).
-Defaults to the `contributors` field in the closest `package.json` upwards from
-the processed [file][vfile], if there is one.
+In Node.js, defaults to the `contributors` field in the closest `package.json`
```

Comment:
if the env isn't Node the default is empty?
What is the default in Deno?

Reply:
🤷‍♂️

Are there `deno` conditions? `browser` fields? I’m not perfectly clear on whether things will work perfectly in Deno for some of these packages